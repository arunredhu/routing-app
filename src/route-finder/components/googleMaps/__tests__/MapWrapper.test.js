import React from 'react';
import renderer from 'react-test-renderer';
import MapWrapperComponent from '../mapWrapper';

describe('MapWrapperComponent Test', () => {
    it('render correctly', () => {
        const geo = {
            startLatLong: [
                "22.372081",
                "114.107877"
            ],
            dropLatLong: [
                "22.284419",
                "114.159510"
            ],
            currLatLong: {
                lat: "29.6512",
                lng: "-82.3426"
            }
        }
        const defaultZoom = 10
        const tree = renderer.create(<MapWrapperComponent defaultZoom={defaultZoom}
            geo={geo} />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});