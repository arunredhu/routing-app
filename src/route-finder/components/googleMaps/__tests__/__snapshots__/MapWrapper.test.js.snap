// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapWrapperComponent Test render correctly 1`] = `
<div
  style={
    Object {
      "height": "400px",
    }
  }
>
  <div
    style={
      Object {
        "height": "100%",
      }
    }
  />
  <div />
</div>
`;
